assets/css/sensei-course-theme/blocks/course-title.scss

```diff
@@ -1,9 +1,9 @@
-:root .wp-block-sensei-lms-course-title {
+.wp-block-sensei-lms-course-title {
+	color: #000000;
```

Comment:
```suggestion
	color: #1E1E1E;
```
According the designs

Reply:
It's black as per the designs I'm following here - https://www.figma.com/file/8mY8ui4zYK1Ylw2yIFi9Lx/Learning-Mode?node-id=20020527-41973&t=wJvNPrH0b40P4Ljq-0

Reply:
Fixed in https://github.com/Automattic/sensei/pull/6783/commits/a829ff887961a4dc85e634966fe827a91c92bd94.